models/articles.model.js

```diff
@@ -45,3 +45,15 @@ exports.selectArticles = () => {
       return rows;
     });
 };
+
+
+exports.selectCommentsByArticleId = (id) => {
+  return db
```

Comment:
Please do some logic first to check whether the article exists in the first place. If the article exists but has no comments, it will return an empty array. However, if the article does not exist, and we say:      
`SELECT * FROM comments WHERE article_id = $1;`,
this won't error out because the id is of a valid type, but it will still return an empty array. By checking whether the article exists and doing Promise.reject if it doesn't, we can feel secure that if we do get 0 rows back - it's definitely because there are no associated comments.